scripts/fetch-fonts.mjs

```
@@ -0,0 +1,193 @@
+    // Latin slice ONLY. Reggae One styles the logo and nav links, which are ASCII.
+    // Its other 122 CJK slices are dead weight.
+    css: 'Reggae+One',
+    slug: 'reggae-one',
+    latinOnly: true,
+  },
+];
+
+const BASIC_LATIN = 'U+0000-00FF';
+
+async function fetchText(url) {
+  const res = await fetch(url, { headers: { 'User-Agent': UA } });
```

ESLint

[error] 'fetch' is not defined.

(no-undef)